### PR TITLE
OBSDOCS-1171: Created a PR for verification of the loki content

### PR DESCRIPTION
**Pull request by shipsing**
**Purpose**: To resolve the issue: https://issues.redhat.com/browse/OBDDOCS-1171 

**Aligned team**: Logging (ODiE)

**Content for preview**:  https://78476--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/logging/logging-6.0/log6x-loki 

**SME review**:  @periklis @xperimental 
**QE review**:  @kabirbhartiRH 
**Peer review**:  @adellape

### CODE REVIEW

File: observability/logging/logging-6.0/log6x-loki.adoc [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+include::modules/logging-loki-retention.adoc[leveloffset=+1]
+
+include::modules/logging-loki-zone-aware-rep.adoc[leveloffset=+1]
+
+include::modules/logging-loki-zone-fail-recovery.adoc[leveloffset=+1]
+
+include::modules/loki-rate-limit-errors.adoc[leveloffset=+1]
+
+include::modules/loki-rbac-rules-permissions.adoc[leveloffset=+1]
+
```
Comment (libander) [inline observability/logging/logging-6.0/log6x-loki.adoc:41]: Eng only needs to verify the include modules above this line. The content below this line is new.

**Review by libander — APPROVED** [review]

File: observability/logging/logging-6.0/log6x-loki.adoc (hunk 1 above)
Comment (shipsing) [reply observability/logging/logging-6.0/log6x-loki.adoc:41]: @libander, can I remove the new content `Loki Architecture: Understanding the Components`  from this PR to avoid any confusion. I will proceed with verification of the moduels only.

Comment (libander) [reply observability/logging/logging-6.0/log6x-loki.adoc:41]: @shipsing - Absolutely. I am 100% onboard with clarity.

**Review by periklis — COMMENTED** [review]
Looks good to me. One issue though persists that channel names in the rendered version are set to `stable-5.9` and the should be `stable-6.0` when Logging 6.0 lands. I am not sure if this is something related to the includes or the following PR:
- https://github.com/openshift/openshift-docs/pull/77407/

File: observability/logging/logging-6.0/log6x-loki.adoc [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+= Storing logs with LokiStack
+include::_attributes/common-attributes.adoc[]
+:context: logging-6x
+
+toc::[]
+
+You can configure a `LokiStack` CR to store application, audit, and infrstructure-related logs. 
+
+== Installation
+Partial includes of modules/logging-loki-cli-install.adoc & logging-loki-gui-install.adoc once https://github.com/openshift/openshift-docs/pull/77407/ is merged.
```
Comment (shipsing) [inline observability/logging/logging-6.0/log6x-loki.adoc:12]: @libander, do we want to include these installation modules or just reference them.
Maybe, we can use a prerequites section for the assembly to mention:
You have installed the Loki Operator by using the CLI or web console. For more details, see cross-references.

**Comment by shipsing** [comment]
@per

> Looks good to me. One issue though persists that channel names in the rendered version are set to `stable-5.9` and the should be `stable-6.0` when Logging 6.0 lands. I am not sure if this is something related to the includes or the following PR:
> 
> * [OBSDOCS-736-4.14 - Logging Installation Fix - cee.next #77407](https://github.com/openshift/openshift-docs/pull/77407)

@periklis, We need to update this detail in the included modules. I will make this change. Thanks

File: modules/logging-identity-federation.adoc [hunk 3]
```diff
@@ -4,18 +4,22 @@
...
 * {product-title} 4.14 and later
 * {logging-uc} 5.9 and later
 
 .Procedure
-* If you use the {product-title} web console to install the {loki-op}, clusters that use short-lived tokens are automatically detected. You are prompted to create roles and supply the data required for the {loki-op} to create a `CredentialsRequest` object, which populates a secret.
 
-* If you use the {oc-first} to install the {loki-op}, you must manually create a subscription object using the appropriate template for your storage provider, as shown in the following examples. This authentication strategy is only supported for the storage providers indicated.
+* Use one of the following options to enable authentication:
+
+** If you use the {product-title} web console to install the {loki-op}, clusters that use short-lived tokens are automatically detected. You are prompted to create roles and supply the data required for the {loki-op} to create a `CredentialsRequest` object, which populates a secret.
```
Comment (shipsing) [inline modules/logging-identity-federation.adoc:18]: Can we provide a YAML example to create a `CredentialsRequest` object for readers' reference?

Comment (libander) [reply modules/logging-identity-federation.adoc:18]: IIRC, Eng's take on this was that having an example of the CredentialsRequest was unnecessary and might be confusing, as it is generated for the user.

File: observability/logging/logging-6.0/log6x-loki.adoc (hunk 2 above)
Comment (libander) [reply observability/logging/logging-6.0/log6x-loki.adoc:12]: xrefs work. :)

File: modules/logging-loki-retention.adoc [hunk 4]
```diff
@@ -26,7 +30,7 @@ metadata:
   name: logging-loki
   namespace: openshift-logging
 spec:
-  limits: 
+  limits:
    global: <1>
       retention: <2>
         days: 20
```
Comment (kabirbhartiRH) [inline modules/logging-loki-retention.adoc:36]: BTW, if we want to target this content on for Logging 6.0, it's better to use v13 schema on the lokistack instead of v11.

File: modules/logging-enabling-loki-alerts.adoc [hunk 5]
```diff
@@ -31,7 +31,7 @@ The `AlertingRule` CR contains a set of specifications and webhook validation de
 
 . Create an `AlertingRule` custom resource (CR):
 +
```
Comment (kabirbhartiRH) [inline modules/logging-enabling-loki-alerts.adoc:33]: The table for tenant application should have something along the lines of 'your application namespace' instead of blank as column 'Valid namespaces for AlertingRule CRs' value.

**Comment by kabirbhartiRH** [comment]
LGTM

File: modules/logging-loki-zone-fail-recovery.adoc [hunk 6]
```diff
@@ -65,21 +65,21 @@ wal-logging-loki-ruler-1
 oc delete pvc __<pvc_name>__  -n openshift-logging
 ----
 +
-. Then delete the pod(s) by running the following command:
+. Delete the pod(s) by running the following command:
 +
 [source,terminal]
 ----
 oc delete pod __<pod_name>__  -n openshift-logging
```
Comment (adellape) [inline modules/logging-loki-zone-fail-recovery.adoc:72]: There are some spots in the preview assembly where commands are either missing their `$` prompt, are using the non-openshift-docs style for user-replaceables (e.g., `__<pvc_name>__`), or both. Please check throughout the new built assembly and adjust the related modules that need fixes (some appear to not be included in this PR yet).

E.g., these ones should look like:
```
$ oc delete pvc <pvc_name> -n openshift-logging
```
and
```
$ oc delete pod <pod_name> -n openshift-logging
```

**Review by adellape — COMMENTED** [review]
Some nice clean-up / fixes in here! 👍

File: modules/logging-enabling-loki-alerts.adoc [hunk 7]
```diff
@@ -17,7 +17,7 @@ The `AlertingRule` CR contains a set of specifications and webhook validation de
 [options="header"]
 |================================================
 | Tenant type    | Valid namespaces for `AlertingRule` CRs
-| application    |
+| application    | `your-application-namespace`
```
[the remark this answers is not in the document]
Comment (shipsing) [reply modules/logging-enabling-loki-alerts.adoc:20]: Your suggestion is on the spot. Thanks
